frontend/public/components/utils/headings.tsx

```diff
@@ -141,7 +145,7 @@ export const PageHeading = connectToModel((props: PageHeadingProps) => {
               )}
             </div>
           )}
-          {!breadcrumbsFor && badge}
+          {!breadcrumbsFor && <span className="co-m-pane__heading-badge">{badge}</span>}
```

Comment:
```suggestion
          {!breadcrumbsFor && badge && ( 
            <span className="co-m-pane__heading-badge">{badge}</span>
          )}
```

Otherwise this his will output a `<span>` regardless, no?

Reply:
Yes, this is erroneous.